tutorials/beginner/decoupling_logic_and_execution.ipynb

```diff
@@ -6,25 +6,25 @@
    "source": [
```

Comment:
<p>There seems to be an extra space here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>